libresourceqt/src/resource-set.cpp

```diff
@@ -35,7 +36,11 @@ ResourceSet::ResourceSet(const QString &applicationClass, QObject * parent,
       audioResource(NULL), autoRelease(initialAutoRelease),
       alwaysReply(initialAlwaysReply), initialized(false), pendingAcquire(false),
       pendingUpdate(false), pendingAudioProperties(false), pendingVideoProperties(false),
+#if (QT_VERSION > QT_VERSION_CHECK(6,0,0))
+      inAcquireMode(false), reqMutex(QRecursiveMutex()), ignoreQ(false)
+#else
```

Comment:
Please just include the differing variables inside #if. inAcquireMode and ignoreQ don't need differences so could be taken out of here. Also the default ctor for the QRecursiveMutex could be skipped.

-> could just have "if < 5.14" without else branch. 

Same on the other ctor.